ceph/rbd/utils.py

```diff
@@ -106,6 +107,45 @@ def check_data_integrity(**kw):
     return 0
 
 
+# def check_data_integrity_for_multiple_images(**kw):
```

Comment:
why commented code here, can you please take this in any other PR?

Reply:
Sure will do.